src/app/clients/create-address/create-address.component.html

```diff
@@ -1,71 +1,71 @@
+        <div fxLayout="row wrap" fxLayoutGap="2%" fxLayout.lt-md="column">
+          <mat-form-field fxFlex="48%">
+            <mat-label>Address Type</mat-label>
+            <mat-select required formControlName="addressTypeID">
+              <mat-option *ngFor="let item of clientAddress?.addressTypeIdOptions" [value]="item?.id">
+                {{ item?.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="newAddressForm.controls.addressTypeID.hasError('required')">
+              Address type is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+          <mat-form-field fxFlex="48%">
+            <mat-label>Street</mat-label>
+            <input matInput formControlName="street">
+          </mat-form-field>
+          <mat-error *ngIf="newAddressForm.controls.street.hasError('required')">
+            Street is <strong>required</strong>
+          </mat-error>
+          <mat-form-field fxFlex="48%">
+            <mat-label>Address Line 1</mat-label>
+            <input matInput formControlName="addressLine1">
+          </mat-form-field>
+          <mat-form-field fxFlex="48%">
+            <mat-label>Address Line 2</mat-label>
+            <input matInput formControlName="addressLine2">
+          </mat-form-field>
+          <mat-form-field fxFlex="48%">
+            <mat-label>Address Line 3</mat-label>
+            <input matInput formControlName="addressLine3">
+          </mat-form-field>
+          <mat-form-field fxFlex="48%">
+            <mat-label>City</mat-label>
+            <input matInput formControlName="city">
+          </mat-form-field>
+          <mat-form-field fxFlex="48%">
+            <mat-label>State</mat-label>
+            <mat-select required formControlName="stateProvinceId">
+              <mat-option *ngFor="let item of clientAddress?.stateProvinceIdOptions" [value]="item?.id">
+                {{ item?.name }}
+              </mat-option>
+            </mat-select>
+          </mat-form-field>
+          <mat-form-field fxFlex="48%">
+            <mat-label>Country</mat-label>
+            <mat-select required formControlName="countryId">
+              <mat-option *ngFor="let item of clientAddress?.countryIdOptions" [value]="item?.id">
+                {{ item?.name }}
+              </mat-option>
+            </mat-select>
+            <mat-error *ngIf="newAddressForm.controls.countryId.hasError('required')">
+              Country is <strong>required</strong>
+            </mat-error>
+          </mat-form-field>
+          <mat-form-field fxFlex="48%">
+            <mat-label>Postal Code</mat-label>
+            <input matInput formControlName="postalCode">
+          </mat-form-field>
+        </div>
+      </mat-card-content>
+      <mat-card-actions fxLayout="row" fxLayout.xs="column" fxLayoutAlign="center" fxLayoutGap="5px">
+        <button type="button" mat-raised-button [routerLink]="['../../view/'+id]">Cancel</button>
```

Comment:
Do not use `+` operator to append the url. Use `,` and specify parameters separately.